eventual-cassandra/src/main/scala/com/evolutiongaming/kafka/journal/eventual/cassandra/EventualCassandraConfig.scala

```diff
@@ -40,41 +40,12 @@ final case class EventualCassandraConfig(
       fetchSize = 1000,
       defaultIdempotence = true)),
   schema: SchemaConfig = SchemaConfig.default,
-  consistencyConfig: EventualCassandraConfig.ConsistencyConfig = EventualCassandraConfig.ConsistencyConfig.default)
+  consistencyConfig: ConsistencyConfig = ConsistencyConfig.default)
 
 object EventualCassandraConfig {
 
   val default: EventualCassandraConfig = EventualCassandraConfig()
 
   implicit val configReaderEventualCassandraConfig: ConfigReader[EventualCassandraConfig] = deriveReader
 
-
-  final case class ConsistencyConfig(
```

Comment:
please at leave empty dummy class with deprecation and a link - where to look for the replacement

Reply:
I fixed it, but I had to rename the class to `CassandraConsistencyConfig` to avoid a naming clash.

Should I do the same in https://github.com/evolution-gaming/kafka-journal/pull/539? :thinking: